exercises/twelve-days/example.go

```diff
@@ -39,9 +39,11 @@ func Verse(i int) string {
 	for from := i; from > 0; from-- {
 		if i != 1 && from == 1 {
 			gifts = fmt.Sprintf("%s, and %s", gifts, verse[wording[from]])
-		} else {
-			gifts = fmt.Sprintf("%s, %s", gifts, verse[wording[from]])
-		}
+		} else if i != from {                                                                                                                   
+      gifts = fmt.Sprintf("%s, %s", gifts, verse[wording[from]])                                                                            
```

Comment:
Let's run `go fmt` within the exercise directory to fix indentation issues for your latest changes.  Once formatted, the travis build should go green.